Source/AwsCommonRuntimeKit/crt/Utilities.swift

```diff
@@ -147,7 +148,7 @@ extension TimeInterval {
         let _millisecond = (self * 1_000).rounded()
         guard _millisecond >= 0 && _millisecond <= Double(UInt32.max) else {
             // todo convert the millisecond conversion errors into aws-crt-swift errors
-            throw CommonRunTimeError.crtError(CRTError(code: AWS_ERROR_INVALID_ARGUMENT.rawValue))
+            throw CommonRunTimeError.crtError(CRTError(code: AWS_CRT_SWIFT_INVALID_ARGUMENT.rawValue))
```

Comment:
We should avoid duplicating errors because they are "common". The distinction between AWS_ERROR_INVALID_ARGUMENT and AWS_CRT_SWIFT_INVALID_ARGUMENT might not be clear to customers, leading to confusion. Having two different errors for the same issue can be unnecessarily complicated.